fv3core/stencils/dyn_core.py

```diff
@@ -292,6 +292,9 @@ def __init__(
             domain=self.grid.domain_shape_full(),
         )
 
+        if self.namelist.rf_fast:
+            self.rayleigh_damping = ray_fast.RayleighDamping(self.grid, self.namelist)
+
```

Comment:
by convention we have been naming these starting with an underscore to indicate they are private to the class, not a huge deal.

Reply:
Good catch